documents/Libraries/pbrlib/genglsl/lib/mx_environment_prefilter.glsl

```diff
@@ -35,14 +35,13 @@ vec3 mx_latlong_map_lookup(vec3 dir, mat4 transform, float lodBias, sampler2D sa
     return vec3(0.0);
 }
 
-vec3 mx_environment_specular(vec3 normal, vec3 view, float roughness)
+vec3 mx_environment_radiance(vec3 N, vec3 V, vec3 X, roughnessinfo roughness, int distribution)
```

Comment:
Use the same name and signature for the two environment sampling methods. Then we can switch the implementation depending on the option.